config/environments/test.rb

```diff
@@ -39,4 +39,9 @@
 
   # Raises error for missing translations
   # config.action_view.raise_on_missing_translations = true
+
+  config.active_job.queue_adapter = :sidekiq
```

Comment:
Maybe, we could move this configuration to `config/application.rb` so is available on all environments (development, test and production).